src/test/java/com/exasol/dbbuilder/DatabaseObjectCreationIT.java

```diff
@@ -6,17 +6,24 @@
 import static com.exasol.dbbuilder.SystemPrivilege.CREATE_SESSION;
 import static com.exasol.dbbuilder.SystemPrivilege.KILL_ANY_SESSION;
 import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.contains;
 import static org.hamcrest.Matchers.equalTo;
 import static org.junit.jupiter.api.Assertions.assertAll;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
 import java.sql.Connection;
 import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 import java.sql.SQLException;
+import java.sql.Statement;
```

Comment:
We don't use * anymore?

Reply:
That's only for class members, not for whole classes.